tests/test_coordinator.py

```
@@ -1861,6 +1863,69 @@ async def test_custom_pulse_overrides_bed_default(
         assert coordinator.motor_pulse_delay_ms == 75
 
 
+class TestRuntimeBedTypeCorrection:
+    """Test runtime Malouf protocol correction after GATT discovery."""
+
+    def _make_coordinator(self, hass, mock_config_entry_data, extra):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_make_coordinator`

(ANN202)